packages/bundler-plugin-core/src/sentry/sourcemaps.ts

```diff
@@ -28,10 +28,13 @@ export function getFiles(includePath: string, includeEntry: InternalIncludeEntry
   if (fileStat.isFile()) {
     files = [{ absolutePath, relativePath: path.basename(absolutePath) }];
   } else if (fileStat.isDirectory()) {
+    const absoluteIgnores = includeEntry.ignore.map((e) => path.join(absolutePath, e));
```

Comment:
Ok, so thinking about this for another round, I might have to swap this to path.join(process.cwd(), ignoreEntry) because that makes more sense w.r.t to what's documented. (also the "node_modules" default only makes sense in that context).